lib/messager.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+require 'date'
+require 'dotenv'
+
+class Messager
+  def initialize
+    @env = Dotenv.load
+  end
+
+  def send_text
+    client = Twilio::REST::Client.new @env["account_sid"],@env["auth_token"]
+
+    friends.each do |key, value|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)